src/components/SiteHeader/index.js

```diff
@@ -2,6 +2,7 @@ import React, { Fragment } from 'react';
 import PropTypes from 'prop-types';
 import { NavLink, withRouter } from 'react-router-dom';
 import styled, { css } from 'styled-components';
+import { ascDefaultTheme } from '@datapunt/asc-core';
```

Comment:
I woudn't use ascDefaultTheme directly. For colors you can use the utility that extracts a certain color from the current theme for instance: `color: ${themeColor('tint', 'level1')}`

Reply:
gedaan